### PR TITLE
Remove overlays from new

### PR DESCRIPTION
**Pull request by crocodoyle**
The following new objects now use a separate page to create objects instead of an overlay:
-DataProviders
-Servers
-Tools
-Sites

Fix for issue #397

### CODE REVIEW

File: BrainPortal/app/views/bourreaux/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+<h2>Add New Server</h2>
+
+<%= error_messages_for @bourreau, :object_name => "server" %>
+
+<%= form_for @bourreau, :as => :bourreau, :url => { :action => "create" }, :datatype => "script" do |f| -%>
+
+  <div class="generalbox">
+
+	<% is_bourreau = @bourreau.is_a?(Bourreau) %>
```
Comment (prioux) [inline BrainPortal/app/views/bourreaux/new.html.erb]: Simplify: the is_bourreau and is_portal checks are no longer necessary, this form will always be with is_bourreau set to true

Comment (prioux) [inline BrainPortal/app/views/bourreaux/new.html.erb]: Also, re-indent, while at it.

File: BrainPortal/app/views/data_providers/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,261 @@
+
```
Comment (prioux) [inline BrainPortal/app/views/data_providers/new.html.erb]: I see two problems with this new file:

* the file `form_fields.html.erb` is no longer needed
* more importantly, much of the embedded doc is obsolete, and a fixed version of it is already in another pull request #393 (installed in a separate file).

**Comment by prioux** [comment]
Andrew, we can't merge this pull request because it would undo some changes that were already commited to the `dev` branch; your created your own branch in your local repo based on an obsolete version of `dev` it seems, and it's missing some pull requests (e.g. #393).

In your lcoal repo you can try to:

* checkout dev
* pull from aces/cbrain's dev to make sure you're up to date
* either re-do all the work done here (including my comments) or try to REBASE your branch to the head of `dev` (that might work).

Also, some more general comments about this PR: some _form_fields.html.erb were left over and also the following models coudl also benefit from this work:

* feedbacks
* groups
* messages

**Comment by crocodoyle** [comment]
How can I find the pages that display feedbacks, groups and messages?

**Comment by crocodoyle** [comment]
Changed overlays for feedbacks, groups and messages, removed the is_bourreau/is_brainportal from data_providers, re-indented. Should be good to go.

**Pull request closed without merging**